HHS/Head-Start-TTADP
# [Prod] Fix bug in goalsForReport query, CSS fixes, add commas to anniversary statistics

## Description of change
There was a bug in the goalsForReport query that was causing extra objectives to be returned from the backend. This PR fixes that bug.

This PR also includes fixes for how the filter pills look on mobile and standardizes vertical spacing around elements, and adds commas to the anniversary statistics for readability

## How to test
Follow the steps on the linked Jira to create a goal and notice there are no extra objectives being loaded

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1598
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1578


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
